### DISCUSSION TITLE
疑似内存溢出org.apache.flink.yarn.YarnTaskExecutorRunner [] - RECEIVED SIGNAL 15: SIGTER

### DISCUSSION THREAD

**Discussion by liuxiaolinlxl** [apache/flink-cdc#597] [Q&A]
现象：关于mongodb cdc的，我们mongodb集群的仲裁节点设置不允许连接，拉数据要求在从库拉取，就配置了这个'connection.options' = 'readPreference=secondaryPreferred'，问题是偶尔报连不上仲裁节点或者xx.xx.xx.xx is no longer member of mongo,报完这类错之后不久，就会被yarn kill掉（报：org.apache.flink.yarn.YarnTaskExecutorRunner [] - RECEIVED SIGNAL 15: SIGTER），请问这咋解决…
![image](https://user-images.githubusercontent.com/44665332/141246113-ece07c00-9199-4e99-a600-d47f589eeb3b.png)

**Comment by Wentao-Zhao** [comment]
我的 mysql cdc 也出现了同样的问题,任务大概跑3小时就会出现这个问题,JobManager报出一个线程冲突的异常
![image](https://user-images.githubusercontent.com/57213057/141292305-66d4edd2-b090-4fe4-86ee-936f6040bd4d.png)

**Reply by yangrong688** [reply]
应该跟这个issue有关，暂时只能增加内存解决一下
https://github.com/ververica/flink-cdc-connectors/issues/865
